core-server/src/main/java/org/orbisgis/orbisserver/control/web/IndexController.java

```diff
@@ -61,46 +71,67 @@
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
 import java.io.DataInputStream;
+import java.io.File;
 import java.io.InputStream;
 import java.util.List;
+import java.util.ArrayList;
 
 /**
  * Very light instance of DefaultController containing a WpsServer.
  *
  * @author Sylvain PALOMINOS
+ * @author Guillaume MANDE
  */
 @Controller
 public class IndexController extends DefaultController {
 
-    /**
-     * Gets the instance of the WpsServer.
-     */
+    /** Gets the instance of the WpsServer. */
     private WpsServer wpsServer = WpsServerManager.getWpsServer();
 
+    /** List all of the OrbisWPS processes into a String which be displayed on the index page. */
+    private String processesList = "";
+
+    /**  Used to display the corresponding xml file to GetCapabilities method. */
+    public WPSCapabilitiesType wpsCapabilitiesType = null;
+
+    /** Used by methods like DescribeProcessRequest to get all the ProcessSummaryType */
+    public List<ProcessSummaryType> processSummaryTypeList = null;
+
+    /** List of all identifier of all the processes */
+    private List<CodeType> codeTypeList = null;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(IndexController.class);
+    private static final I18n I18N = I18nFactory.getI18n(IndexController.class);
+
     /**
      * Injects a template named 'index'.
      */
     @View("index")
-    Template index;
+    public Template index;
```

Comment:
I have all units Tests in the right place now and IT Tests location does not change, they are integration tests which check if the returned page after a http request has the correct content, so there is no specific package for them.